workspaces/demo-app-contracts/package.json

```diff
@@ -12,8 +12,10 @@
     "source"
   ],
   "scripts": {
-    "build": "yarn compile && node ./tools/build.js",
```

Comment:
I would personally keep the compilation of the Solidity contracts in the 'build' step. Now a developer has to know about both a 'build' and a 'compile' step that he has to perform.

Reply:
As discussed, we will keep it for the moment.